app/models/bookings/booking.rb

```diff
@@ -36,6 +36,8 @@ class Booking < ApplicationRecord
       errors.add :date, :not_changed unless date_changed?
     end
 
+    validates :experience_type, presence: true
```

Comment:
Could we validate this is either virtual or inschool? `inclusion: { in: ["virtual", "inschool"] }`

Reply:
I've added the inclusion validation :+1: